### PR TITLE
Included Database Management Courses

### PR DESCRIPTION
**Pull request by HarshRanaOC**
## What does this PR do?
Add resource(s) |  Improve repo

### Why is this valuable (or not)?
Yes
### How do we know it's really free?
Yes
### For book lists, is it a book? For course lists, is it a course? etc.
Course
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -534,9 +534,12 @@
 * [15-721 Advanced Database Systems (Spring 2023)](https://www.youtube.com/playlist?list=PLSE8ODhjZXjYzlLMbX3cR0sxWnRM7CLFn) - CMU Database Group
 * [CMU Intro to Database Systems](https://youtube.com/playlist?list=PLSE8ODhjZXjaKScG3l0nuOiDTTqpfnWFf) - CMU Database Group
 * [Database Design and Management](https://www.udemy.com/course/database-design-and-management) - Visual Paradigm (Udemy)
+* [Database Management System | CS & IT | GATE 2024](https://www.youtube.com/playlist?list=PLPvaSRcEQh4kfVIyezAQu9Mvj5FBk_OEN) - GATE Wallah (English)
```
Comment (sohansai) [inline courses/free-courses-en.md:537]:
```suggestion
* [Database Management System \| CS & IT \| GATE 2024](https://www.youtube.com/playlist?list=PLPvaSRcEQh4kfVIyezAQu9Mvj5FBk_OEN) - Gate Wallah (English)
```

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -534,9 +534,12 @@
...
+* [Database Management Systems](https://www.youtube.com/playlist?list=PLZ2ps__7DhBYc4jkUk_yQAjYEVFzVzhdU) - IIT Madras - B.S. Degree Programme
```
Comment (sohansai) [inline courses/free-courses-en.md:538]:
```suggestion
* [Database Management Systems](https://www.youtube.com/playlist?list=PLZ2ps__7DhBYc4jkUk_yQAjYEVFzVzhdU) - IIT Madras B.S. Degree Programme
```

**Comment by HarshRanaOC** [comment]
Hello @sohansai , I have made the rquired changes.
Regards

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -534,9 +534,12 @@
 * [15-721 Advanced Database Systems (Spring 2023)](https://www.youtube.com/playlist?list=PLSE8ODhjZXjYzlLMbX3cR0sxWnRM7CLFn) - CMU Database Group
 * [CMU Intro to Database Systems](https://youtube.com/playlist?list=PLSE8ODhjZXjaKScG3l0nuOiDTTqpfnWFf) - CMU Database Group
 * [Database Design and Management](https://www.udemy.com/course/database-design-and-management) - Visual Paradigm (Udemy)
+* [Database Management System \| CS & IT \| GATE 2024](https://www.youtube.com/playlist?list=PLPvaSRcEQh4kfVIyezAQu9Mvj5FBk_OEN) - Gate Wallah (English)
```
Comment (eshellman) [inline courses/free-courses-en.md:537]:
```suggestion
* [Database Management System \| CS & IT \| GATE 2024](https://www.youtube.com/playlist?list=PLPvaSRcEQh4kfVIyezAQu9Mvj5FBk_OEN) - Gate Wallah English
```
The parentheses get parsed as access notes

File: courses/free-courses-en.md [hunk 4]
```diff
@@ -534,9 +534,12 @@
...
+* [Database Management Systems](https://www.youtube.com/playlist?list=PLZ2ps__7DhBYc4jkUk_yQAjYEVFzVzhdU) - IIT Madras B.S. Degree Programme
```
Comment (eshellman) [inline courses/free-courses-en.md:538]:
```suggestion
* [Database Management Systems](https://www.youtube.com/playlist?list=PLZ2ps__7DhBYc4jkUk_yQAjYEVFzVzhdU) - Partha Pratim Das (IIT Madras B.S. Degree Programme)
```
Please add author names where appropriate.

**Review by eshellman — CHANGES_REQUESTED** [review]

**Comment by HarshRanaOC** [comment]
Hello @eshellman,
I have implemented the changes as per your suggestions.
Regards.

File: courses/free-courses-hi.md [hunk 5]
```diff
@@ -157,6 +159,7 @@
 * [Blockchain Full Course in Hindi](https://www.youtube.com/playlist?list=PLRlT7xBRpp9MiMN25XJjUVz01rGFQohq2) - Innovate India
 * [Complete Blockchain Development Course for Beginners in Hindi](https://www.youtube.com/watch?v=RkYVVC2vXho) - web3Mantra
 * [Ethereum](https://www.youtube.com/playlist?list=PL-Jc9J83PIiE3QA0h3I6HDYNXejdPFKFN) - Pepcoding
+* [Playlist to Blockchain 🔥🔥](https://www.youtube.com/playlist?list=PLPIwNooIb9vgfXs-QkRYqqZbDXX-yLf59) - Perfect Computer Engineer
```
Comment (eshellman) [inline courses/free-courses-hi.md:162]: no emoji
```suggestion
* [Playlist to Blockchain](https://www.youtube.com/playlist?list=PLPIwNooIb9vgfXs-QkRYqqZbDXX-yLf59) - Perfect Computer Engineer
```

File: courses/free-courses-hi.md [hunk 6]
```diff
@@ -133,7 +133,9 @@
 
 ### Artificial Intelligence
 
+* [Artificial Intelligence (Complete Playlist)](https://www.youtube.com/playlist?list=PLxCzCOWd7aiHGhOHV-nwb0HR5US5GFKFI) - Gate Smashers
 * [Generative AI Series - Showcasing Generative AI By Building Projects](https://www.youtube.com/playlist?list=PLu0W_9lII9aiS4rUVp2jXwIvCruo27sG6) - CodeWithHarry
+* [Playlist to Artificial Intelligence 🔥🔥](https://www.youtube.com/playlist?list=PLPIwNooIb9vgB1DQEftkKA3qOdeC4vonA) - Perfect Computer Engineer
```
Comment (eshellman) [inline courses/free-courses-hi.md:138]:
```suggestion
* [Playlist to Artificial Intelligence ](https://www.youtube.com/playlist?list=PLPIwNooIb9vgB1DQEftkKA3qOdeC4vonA) - Perfect Computer Engineer
```

**Review by eshellman — CHANGES_REQUESTED** [review]

**Comment by eshellman** [comment]
Hacktoberfest give a few days after the month to complete a PR. It's still October here!

**Pull request merged**